tso-server/src/main/java/com/yahoo/omid/tso/PersistenceProcessorImpl.java

```diff
@@ -395,7 +266,13 @@ static void makePersistTimestamp(PersistEvent e, long startTimestamp, Channel c,
             e.channel = c;
             e.monCtx = monCtx;
         }
-        
+
+        static void makePersistLowWatermark(PersistEvent e, long lowWatermark, MonitoringContext monCtx) {
```

Comment:
Changed to local initialization functions. Constructors can't be used in this case because PersistEvents are allocated only at Batch allocation and reuse to save gc time.